src/components/Footer.jsx

```diff
@@ -34,39 +33,67 @@ const Faq = styled.a`
   text-decoration: none;
 `;
 
+const SnsList = styled.div`
+  display: flex;
+  gap: 12px;
+  align-items: center;
+`;
+
 const Sns = styled.div`
   display: flex;
   gap: 12px;
 `;
 
+const snsList = [
```

Comment:
상수의 경우 대문자로 적어주는 게 일반적인 거 같아요!
```suggestion
const SNS_LIST = [
```